styles/styles.css

```diff
@@ -0,0 +1,368 @@
+	color: #0e4769;
+}
+
+.our-dogs-title {
+	margin: 3rem 0 3rem 0;
+}
+
+
+/* HEADER STYLING */
+
+header {
+	background-image: url("../images/header-image.jpg");
+	background-color: black;
+	background-size: cover;
+}
+
+.header-top {
+	margin: 0 1.5rem 4rem 1.5rem;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	flex-wrap: wrap;
+	;
+}
+
+.logo-bundle {
+	margin-top: 1.5rem;
+	margin-bottom: 1.5rem;
+}
+
+.shopping-cart-icon {
+	width: 1.5rem;
+}
+
+.logo-bundle p {
+	color: #2cc749;
+	font-family: "Lora", serif;
+	font-weight: 700;
+	font-style: italic;
+	font-size: 1.5rem;
+	margin: 0;
+	margin-right: 1rem;
+	padding: 0;
+}
+
+.header-logo {
+	width: 80%;
+}
+
+.shopping-cart p {
+	display: inline-block;
+	color: white;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+}
+
+.cart-button {
+	background-color: #2cc749;
+	border: none;
+	color: white;
+	padding: 1rem 2.5rem;
+	text-align: center;
+	text-decoration: none;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+	font-size: 1rem;
+	display: block;
+}
+
+
+/* HEADER STYLING: NAVIGATION */
+
+.header-nav {
+	background: #E4E4E4;
+	list-style: none;
+	padding: .5rem 0;
+}
+
+.header-nav li {
+	display: inline-block;
+	margin: 1rem 0;
+}
+
+.header-nav a {
+	text-decoration: none;
+	color: #253979;
+	font-family: "Open Sans", sans-serif;
+	font-weight: 700;
+	font-size: 1rem;
+	padding: 1rem 2rem;
+}
+
+.header-nav a:hover {
+	background-color: #253979;
+	color: white;
+}
+
+
+/* FOOTER STYLING */
+
+footer {
+	background-color: #253979;
+}
+
+.footer-elements {
+	padding: 4rem;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.footer-logo {
+	width: 45%;
+}
+
+.footer-nav {
+	padding: 0;
+}
+
+.footer-nav li {
+	font-family: "Open Sans", sans-serif;
+	color: white;
+	display: inline-block;
+	margin: 1rem;
+}
+
+.footer-nav a {
+	text-decoration: none;
+	color: white;
+}
+
+.footer-elements p {
+	font-family: "Open sans", sans-serif;
+	font-size: 1rem;
+	width: 60%;
+	color: white;
+	margin-bottom: 0;
+	text-align: center;
+}
+
+small {
+	color: white;
+}
+
+
+/* INDEX CSS */
+
+main {
+	margin: 1rem 10rem;
+}
+
+.about-us {
+	margin-bottom: 6rem;
+	margin-top: 3rem;
+	display: grid;
+	/* grid-template-columns: 1fr 1fr; */
+	grid-template-columns: minmax(300px, .5fr) 1fr;
+	gap: 2.5rem;
+}
+
+.about-us-blurb {
+	align-self: center;
+	justify-self: center;
+}
+
+.about-us-blurb p {
+	font-size: 1.4rem;
+}
+
+
+/* DOG CARD CSS */
+
+.dog-card-board {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+	gap: 3rem;
+}
+
+.od-card-board {
+	margin-bottom: 4rem;
+}
+
+.dog-card {
+	border: 2px solid #e8e8e8;
+	border-radius: 3px;
+	box-shadow: 0px 0px 10px -1px rgba(0, 0, 0, 0.20);
+}
+
+.dog-card-details {
+	padding: 1rem;
+}
+
+.dog-card>h3 {
+	margin-left: 1rem;
+}
+
+.dog-card-details>p:nth-of-type(1) {
+	display: inline;
+	font-weight: 700;
+}
+
+.dog-card-details>p:nth-of-type(2) {
```

Comment:
these n-th of type might get complicated fast. Maybe instead use a class for that element